randovania/games/samus_returns/logic_database/Area 1.txt

```diff
@@ -16,19 +16,25 @@ Extra - asset_id: collision_camera_000
           All of the following:
               Morph Ball
               Any of the following:
-                  # https://youtu.be/NzXfFnGwZkc
-                  Melee Clip (Intermediate)
+                  # Melee Clip: https://youtu.be/NzXfFnGwZkc
+                  Melee Clip (Intermediate) and Out of Bounds Movement (Intermediate)
                   All of the following:
                       Any of the following:
                           # Dealing with the crumbles
                           Spider Ball or Enabled Remove Area 1 Crumble Blocks
                           # https://youtu.be/eVkMBrWB0Wk
                           Phase Drift Skip (Advanced)
+                          All of the following:
+                              # Phase Drift
+                              Phase Drift
+                              Mid-Air Morph (Beginner) or Movement (Beginner) or Lay Bomb or Use Spring Ball
                       Any of the following:
                           # Reaching the elevator
                           Power Bomb or Screw Attack
-                          # Break the shot blocks by using Free Aim while crouching
-                          Shoot Any Missile
+                          All of the following:
+                              Shoot Any Missile
+                              # Break the shot blocks
+                              Wave Beam or Precise Aiming (Beginner) or Lay Any Bomb
```

Comment:
Should we consider the non progressives case for the other beams?

Reply:
easier not to.